src/system_window.cpp

```diff
@@ -568,12 +572,14 @@ void SystemWindow::EndFrame()
 
 		glEnable( GL_TEXTURE_2D );
 
+		#ifndef __APPLE__
```

Comment:
I suggest to wrap the whole branch with `use_gl_context_for_software_renderer_` into conditional compilation.
```cpp
else if( use_gl_context_for_software_renderer_ )
{
#ifndef __APPLE__
...
#endif
}```

Reply:
fixed